frontend/views/containers/settings/UserProfile.vue

```diff
@@ -1,82 +1,63 @@
 <template lang='pug'>
   .settings-container
-    p.username @{{ ourUsername }}
+    span.c-username @{{ ourUsername }}
+
+    form(@submit.prevent='')
+      .c-avatar
+        label.c-avatar-field
+          avatar.c-avatar-img(
+            :src='attributes.picture'
+            ref='picture'
+          )
+          i18n.link.c-avatar-text Change avatar
 
-    section.card
-      form(
-        ref='ProfileForm'
-        name='ProfileForm'
-        @submit.prevent='save'
-      )
-        p.is-success(
-          v-if='ephemeral.profileSaved'
-          data-test='profileSaveSuccess'
-        )
-          i.icon-check
-          i18n Profile saved successfully!
-
-        p.error(v-if='ephemeral.errorMsg') {{ ephemeral.errorMsg }}
-
-        .avatar(:class='{ error: $v.form.picture.$error }')
-          label(for='profilePicture')
-            avatar(
-              :src='userPictureInitial'
-              ref='picture'
-            )
-            i18n.link Change avatar
-
-          input.profilePictureInput#profilePicture(
+          input.sr-only(
             type='file'
             name='profilePicture'
             accept='image/*'
             @change='fileChange($event.target.files)'
             placeholder='http://'
             data-test='profilePicture'
           )
+        // TODO #658
+        banner-scoped(ref='pictureMsg' data-test='pictureMsg' class='c-pictureMsg')
```

Comment:
Usually instead of `class='c-pictureMsg'` the codebase does `banner-scoped.c-pictureMsg`. Is there a reason that was not done here? If so, can it be documented in a comment to explain why?

Also, out of curiosity, which tag is  the `c-pictureMsg` applied to inside of `BannerScoped.vue`?

Reply:
changed to `banner-scoped.c-pictureMsg`, it was a typo.

When passing a class on a component, that class is _always_ automatically appended to the parent element of that component, unless explicit said otherwise using `$attrs`.